src/smokeTest/java/uk/gov/hmcts/futurehearings/hmi/smoke/HmiApiSmokeTest.java

```diff
@@ -35,4 +36,11 @@ public void initialiseValues() throws Exception {
         super.initialiseValues();
         setRootContext(hmiApiRootContext);
     }
+
+    @Test
+    @DisplayName("Smoke Test to test the root endpoint")
+    void testRootHmiApiGet() {
+        assertEquals(HttpStatus.OK.value(), RestClient.makeGetRequest(getHeadersAsMap(),
```

Comment:
Might be nice to still separate out the response so it would be like.
`response = RestClient.makeGetRequest(getHeadersAsMap(), getAuthorizationToken(), getRootContext()).getStatusCode())`

`assertEquals(HttpStatus.OK.value(), response)`

Not that worried about this one though so up to you